tests/acceptance/features/bootstrap/SearchContext.php

```diff
@@ -62,8 +65,13 @@ public function userSearchesUsingWebDavAPI(
 		$body
 			= "<?xml version='1.0' encoding='utf-8' ?>\n" .
 			"	<oc:search-files xmlns:a='DAV:' xmlns:oc='http://owncloud.org/ns' >\n" .
-			"		<oc:search>\n" .
-			"			<oc:pattern>$pattern</oc:pattern>\n";
+			"		<oc:search>\n";
+		if ($scope !== null) {
+			$user_id = $this->featureContext->getUserIdByUserName($user);
+			$body .= "			<oc:pattern>*$pattern* scope:$user_id$scope</oc:pattern>\n";
+		} else {
+			$body .= "			<oc:pattern>$pattern</oc:pattern>\n";
```

Comment:
spaces should not matter
```suggestion
			$body .= "<oc:pattern>*$pattern* scope:$user_id$scope</oc:pattern>\n";
		} else {
			$body .= "<oc:pattern>$pattern</oc:pattern>\n";
```